test/L1ERC20Bridge.t.sol

```diff
@@ -3,33 +3,159 @@ pragma solidity ^0.8.0;
 
 import {Test} from "forge-std/Test.sol";
 import {IGovernor} from "openzeppelin/governance/Governor.sol";
+import {WormholeRelayerBasicTest} from "wormhole-solidity-sdk/testing/WormholeRelayerTest.sol";
 
+import {L1Block} from "src/L1Block.sol";
 import {IERC20Mint} from "src/interfaces/IERC20Mint.sol";
 import {FakeERC20} from "src/FakeERC20.sol";
 import {L1ERC20Bridge} from "src/L1ERC20Bridge.sol";
+import {L2ERC20} from "src/L2ERC20.sol";
 import {Constants} from "test/Constants.sol";
 import {GovernorMock} from "test/mock/GovernorMock.sol";
 
-contract L1ERC20BridgeTest is Test, Constants {
-  IERC20Mint erc20;
+contract L1ERC20BridgeHarness is L1ERC20Bridge {
+  constructor(
+    address _token,
+    address _relayer,
+    address _governor,
+    uint16 _sourceId,
+    uint16 _targetId
+  ) L1ERC20Bridge(_token, _relayer, _governor, _sourceId, _targetId) {}
 
-  function setUp() public {
-    vm.createSelectFork(vm.rpcUrl("avalanche_fuji"));
-    erc20 = new FakeERC20("Hello", "WRLD");
+  function withdraw(address account, uint256 amount) public {
```

Comment:
I don't think we are inheriting the base contracts in the tests.  I went with the test harness approach because it was recommended by foundry.

https://book.getfoundry.sh/tutorials/best-practices?highlight=test%20harnin#test-harnesses

Reply:
Got it, makes sense. I misread this, thanks for the explanation 👍